connection_test.go

```diff
@@ -167,27 +167,27 @@ var _ = Describe("Connection", func() {
 		Expect(err).ToNot(HaveOccurred())
 		defer connection.Close()
 		_, _, err = connection.Tokens()
-		Expect(transport.called).To(BeTrue())
+		Expect(transport.called > 2).To(BeTrue()) // it means the retry was called
```

Comment:
If retry isn't called, will the test fail?

Reply:
of course